src/mmw/apps/modeling/tasks.py

```diff
@@ -332,10 +332,14 @@ def change_key(modification):
 
 @shared_task
 def run_gwlfe(model_input):
-    # TODO pass real input to model instead of reading it from gms file
-    gms_filename = join(dirname(abspath(__file__)),
-                        'mapshed/data/sample_input.gms')
-    gms_file = open(gms_filename, 'r')
-    z = parser.GmsReader(gms_file).read()
+    z = parser.DataModel(model_input)
+    output = StringIO()
+    writer = parser.GmsWriter(output)
+    writer.write(z)
+
+    output.seek(0)
+
+    reader = parser.GmsReader(output)
+    z = reader.read()
```

Comment:
May want to add a comment explaining the point of this.

It's kind of a hack, but the alternative is spending more time moving defaults from the parser to the DataModel.